<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/tools.png</normaloff>:/icons/tools.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabServer">
      <attribute name="title">
       <string>Connection</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelServer">
         <property name="text">
          <string>&amp;IRC server</string>
         </property>
         <property name="buddy">
          <cstring>lineEditServer</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditServer"/>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="labelPort">
         <property name="text">
          <string>&amp;Port</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxPort</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QSpinBox" name="spinBoxPort">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>6667</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelNick">
         <property name="text">
          <string>&amp;Nick</string>
         </property>
         <property name="buddy">
          <cstring>lineEditNick</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditNick"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelUserName">
         <property name="text">
          <string>&amp;User name</string>
         </property>
         <property name="buddy">
          <cstring>lineEditUsername</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditUsername"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelRealName">
         <property name="text">
          <string>&amp;Real name</string>
         </property>
         <property name="buddy">
          <cstring>lineEditRealName</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditRealName"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelPassword">
         <property name="text">
          <string>P&amp;assword</string>
         </property>
         <property name="buddy">
          <cstring>lineEditPassword</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelCharset">
         <property name="text">
          <string>&amp;Charset</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxCharset</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBoxCharset"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelAutoJoin">
         <property name="text">
          <string>Auto-join to channels:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBoxAutoConnect">
         <property name="text">
          <string>C&amp;onnect automatically</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QPushButton" name="pushButtonAutoJoin">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInterface">
      <attribute name="title">
       <string>Interface</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBoxHideConsole">
         <property name="text">
          <string>&amp;Hide &quot;Console&quot; tab</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBoxTrayIcon">
           <property name="text">
            <string>&amp;Tray icon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxMinimizeToTray">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Minimize to tray when closed</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxFonts">
         <property name="title">
          <string>Fonts</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Console font</string>
            </property>
            <property name="buddy">
             <cstring>lineEditConsoleFont</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditConsoleFont">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="toolButtonConsoleFont">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>C&amp;hat font</string>
            </property>
            <property name="buddy">
             <cstring>lineEditChatFont</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditChatFont">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="toolButtonChatFont">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxTimeStamps">
         <property name="title">
          <string>Time stamps</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxTimeStamps">
            <property name="text">
             <string>&amp;Enable time stamps</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelFormat">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Format</string>
            </property>
            <property name="buddy">
             <cstring>lineEditFormat</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditFormat">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string notr="true">[hh:mm:ss]</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSounds">
      <attribute name="title">
       <string>Sounds</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBoxSoundsEnabled">
         <property name="text">
          <string>&amp;Enable sounds</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SoundsTable" name="tableSounds">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SoundsTable</class>
   <extends>QTableView</extends>
   <header>soundstable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTrayIcon</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxMinimizeToTray</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTimeStamps</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelFormat</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTimeStamps</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditFormat</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxSoundsEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>tableSounds</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
